app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/ContributorsFragment.java

```diff
@@ -59,7 +59,7 @@ public void refreshView(State state) {
         final Product product = stateFromActivity.getProduct();
         if (isNotBlank(product.getCreator())) {
             String[] createdDate = getDateTime(product.getCreatedDateTime());
-            String creatorTxt = getString(R.string.creator_history, createdDate[0], createdDate[1]);
+            String creatorTxt = getString(R.string.creator_history, createdDate[0], createdDate[1],product.getCreator());
             creatorText.setMovementMethod(LinkMovementMethod.getInstance());
             creatorText.setText(creatorTxt + " ");
             creatorText.append(getContributorsTag(product.getCreator()));
```

Comment:
There is no need to append the contributor's name anymore. This line can be removed.